src/android/io/branch/BranchSDK.java

```diff
@@ -41,44 +41,32 @@
      */
     public BranchSDK()
     {
+
         this.activity = null;
         this.instance = null;
         this.branchObjectWrappers = new ArrayList<BranchUniversalObjectWrapper>();
-    }
 
-    /**
-     * Called when the activity receives a new intent.
-     */
-    public void onNewIntent(Intent intent)
-    {        
-        this.activity = this.cordova.getActivity();
-        this.activity.setIntent(intent);
-
-        if (this.activity != null) {
-            this.initSession(null);
-        }
     }
 
     /**
-     * Called when the activity will start interacting with the user.
-     *
-     * @param multitasking A {@link boolean} flag indicating if multitasking is turned on for app
+     * Called after plugin construction and fields have been initialized.
      */
     @Override
-    public void onResume(boolean multitasking) {
+    protected void pluginInitialize() {
```

Comment:
According to Cordova: "Called after plugin construction and fields have been initialized."